code/datums/gamemode/factions/bloodcult/bloodcult.dm

```diff
@@ -523,12 +523,48 @@ var/veil_thickness = CULT_PROLOGUE
 			else
 				data[BLOODCOST_LID_CONTAINER] = 1
 
+	var/mob/living/silicon/robot/robot_user = user
+	if(istype(robot_user)) 
+		var/module_items = list(robot_user.module_state_1,robot_user.module_state_2,robot_user.module_state_3) //This function allows robot modules to be used as blood sources. Somewhat important, considering silicons have no blood.
```

Comment:
Why would you do this? Use robot_user.get_all_slots() or robot_user.get_equipped_items()

Reply:
I'm not too familiar with the robot helpers, so I will implement these to ungunk it. Thanks for the tips.